### ISSUE TITLE
TypeORM is ignoring the `type` field when set on a PrimaryGeneratedColumn

### ISSUE THREAD

**Issue by voodooattack** [typeorm/typeorm#778]
I have this setup:
```ts
@Entity()
export class Message {
  @PrimaryGeneratedColumn('increment', { type: 'bigint' })
  id: number;
}
```
This results the following schema:
```
server_1    | executing query: CREATE TABLE "message" ("id" SERIAL NOT NULL, PRIMARY KEY("id"))
```

This is causing the database table to have an `integer` type for the `id` field, which is not ideal for my application.

Shouldn't TypeORM use `bigserial` in this case?

**Comment by pleerock** [comment]
Its something we need to add support to. Im on vacation this month, this will be implemented next month.

**Comment by voodooattack** [comment]
@pleerock No worries. It can wait. :)

**Comment by aufula** [comment]
wait too :)

**Comment by AlexMesser** [comment]
fixed and will be released in `0.1.0-alpha.48`
